nova/utils.py

```diff
@@ -269,7 +269,9 @@ def generate_uid(topic, size=8):
 # ~6 bits per symbol
 DEFAULT_PASSWORD_SYMBOLS = ('23456789',  # Removed: 0,1
                             'ABCDEFGHJKLMNPQRSTUVWXYZ',   # Removed: I, O
-                            'abcdefghijkmnopqrstuvwxyz')  # Removed: l
+                            'abcdefghijkmnopqrstuvwxyz',  # Removed: l
+                            CONF.password_additional_symbols,
+                           )
```

Comment:
If you're editing this anyway, you could move it below the next function close to `generate_password()`. `last_completed_audit_period()` was obviously inserted sometime later and the connection overlooked.

Reply:
Sure, let's hope upstream doesn't see it the other way around :)